src/datadog/remote_config.h

```diff
@@ -44,15 +45,16 @@ class RemoteConfigurationManager {
   };
 
   TracerSignature tracer_signature_;
-  ConfigManager& config_manager_;
-  std::string client_id_;  ///< Identifier a `RemoteConfigurationManager`
```

Comment:
?

Reply:
I figured the comment contained a typo:
```
///< Identifier a `RemoteConfigurationManager`
```
I'm not sure what the extra `/>` is doing, and "Identifier" should probably be "Identifies".

Ok, suppose the comment is "Identifies a `RemoteConfigurationManager`." True enough, but that doesn't really help me understand what `client_id_` is or is for. So, I thought about how best to describe `client_id_` in a comment. I didn't think of anything that was better than nothing, so I went with nothing.